.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 Google LLC
```

Comment:
Can you fork this file from the build and deploy yml? Most of the contents are the same right?

Reply:
Done.